data/en/1_0005.json

```diff
@@ -151,12 +151,12 @@
                             "description": "If possible, please include:",
                             "list": ["the percentage increase of new pay rates",
                                 "the date your figures are back dated from",
-                                "the percentage or number of employees who received a new pay rate"
```

Comment:
Ah, I ran this change past Charlotte in person and didn't update the copy document. Sorry, my bad :(